mobSpawnerTests.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * tests for the spawner object
+ */
+
+import { Spawner } from './src/classes/spawner';
+class spawnerTester{

```

Comment:
Classnames should be instantiated
```suggestion
class SpawnerTester {
```